src/dw_design_system/dwds/components/author.scss

```
@@ -0,0 +1,20 @@
+.dwds-author {
+    &.content-sidebar {
+        --sidebar-gap: 0;
+        --sidebar-size: 60px;
+        --sidebar-min-size: 50%;
+        align-items: center;
+
+        > div:first-child {
+            margin: var(--s-4);
+            text-align: center;
+            display: flex;
+            justify-content: center;
+
+            a:first-child {
```

There's only one `a` tag below the div so we can just say `> a` which will target only the `a` tags DIRECTLY below the first `div` under the `.content-sidebar`

```suggestion
            > a {
```